docs/models.md

```diff
@@ -0,0 +1,80 @@
+## Models
+
+Model_Name            | Description
+----------------------|------------------------------------------------------------------------------------------------------
+Subject               | Used to store *subject_name* and its corresponding *paper_id*.
+Teacher               | Used to store *teacher_name* and their *department*.
+TeacherSubjectMapping | Used to store the realationship between Subject and Teacher model. Eg. X teacher teach Y subject.
+Section               | It store the seciton, branch, year, shift and Teacher -----> Subject record of a particular class.
+Feedback	      | Used to store feedback of every teacher on the basis of the subject he/she teach.
+
+## Model's Field
+
+**1. Subject**
+
+Field_Name       | Description
+-----------------|--------------------------------------------------------------------
+**subject_name** | This field is used to store all subject name in the databse.
+**paper_id**     | This field is used to store paper id of each subject in the databse.
+
+**2. Teacher**
+
+Field_Name      | Description
+----------------|---------------------------------------------
+**teacher_name**| It store every teacher name in the database.
+**teacher_dept**| It store department name of each teacher in the database.
+
+**3. TeacherSubjectMapping**
+
+Field_Name      | Description
+----------------|--------------------------------------------------------------------------------------------
+**teacher_name**| It acts like a foreign key and input values from the teacher_name field of Teacher model.
+**subject_name**|It also acts like a foreign key and input values from the subject_name field of Subject model.
+
+**4. Feedback**
+
+*Feedback model's field are the characteristics of teacher. Teachers get rating on the basis of their characteristic min = 1
+ and max = 5.*
+ 
+- **Punctuality**
+- **Subject_knowledge**
+- **Behaviour**
+- **Method_of_teaching**
+- **Audibility**
+- **Syllabus_coverage**
```

Comment:
Format to describe each model should be consistent. So if you are following tabular method. Keep it same.